core/postfix/Dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM alpine:3.8
 
 RUN apk add --no-cache postfix postfix-pcre rsyslog \
-    python3 py3-pip \
+ && python3 py3-pip \
  && pip3 install --upgrade pip
```

Comment:
You need another backslash at the end of this line.

Reply:
God do I need more coffee.